dev/fattest.simplicity/src/componenttest/annotation/AllowedFFDC.java

```diff
@@ -21,11 +21,13 @@
 public @interface AllowedFFDC {
 
     public static final String ALL_FFDC = "ALL_FFDC";
+    public static final String ALL_REPEAT_ACTIONS = "ALLREPEATACTIONS";
```

Comment:
can we make this constant value separate by underscores so it's a bit more readable?